app/src/main/java/org/luxoft/sdl_core/MainActivity.java

```diff
@@ -67,8 +68,10 @@ protected void onCreate(Bundle savedInstanceState) {
         start_sdl_button = findViewById(R.id.start_sdl_button);
         stop_sdl_button = findViewById(R.id.stop_sdl_button);
 
-        start_sdl_button.setEnabled(true);
-        stop_sdl_button.setEnabled(false);
+        Intent intent = getIntent();
```

Comment:
@AKalinich-Luxoft I have found another way: add PendingIntent.FLAG_IMMUTABLE to the PendingIntent. In future we can change launchMode of MainActivity to prevent similar problems